### DISCUSSION TITLE
problema al ejecutar la el main de market

### DISCUSSION THREAD

**Discussion by llambias** [IIC2133-PUC/2024-1#79] [Ayuda] [answered]
![Captura de pantalla 2024-04-17 155821](https://github.com/IIC2133-PUC/2024-1/assets/80849968/d5bbbb98-6cf8-4181-b784-fb756450fa75)

no he logrado compilar el archivo market/main.c y sigo obteniendo que o el archivo no existe o no puede procesar binario

**Comment by PabloNavarroY** [comment, marked as answer]
![IMG_20240417_170624.jpg](https://github.com/IIC2133-PUC/2024-1/assets/155672565/f609a3c0-0137-42f8-9fa6-592db55d0214)
